rosa/cli.py

```diff
@@ -17,7 +17,10 @@ class CommandExecuteError(Exception):
 
 
 class NotLoggedInError(Exception):
-    pass
+    def __init__(self):
+        super().__init__(
+            "Not logged in to OCM, either pass 'token' or log in before running."
```

Comment:
Please remove the msg from here and let the caller send it
This will not fit when raising it from line 34 since you already send the token